pages/imprint.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div
+    class="flex h-screen flex-col items-center overflow-scroll bg-blue-1-light px-24 py-16 text-xl leading-relaxed dark:bg-blue-1-dark"
+  >
+    <p class="text-3xl font-semibold">Impressum</p>
+    <div class="my-8">
+      <p>
+        <b
+          >Informationen und Offenlegung gemäß &sect;5 (1) ECG, &sect; 25
+          MedienG, &sect; 63 GewO und &sect; 14 UGB</b
+        >
+      </p>
+      <p><b>Webseitenbetreiber:</b> Marco Janderka</p>
+      <p><b>Anschrift:</b> Hochmayrstraße 44, 2384 Breitenfurt bei Wien</p>
+      <p>
+        <b>Kontaktdaten:</b> <br />
+        Telefon: +43 664 3975756 <br />
+        Email: marco.janderka@htl.rennweg.at
+      </p>
+      <p>
+        <b>Anwendbare Rechtsvorschrift: </b>
+        <a
+          href="https://www.ris.bka.gv.at/"
+          class="underline decoration-solid underline-offset-4"
+          >www.ris.bka.gv.at </a
+        ><br />
+      </p>
+      <p>
+        <b>Online Streitbeilegung:</b> Verbraucher haben die Möglichkeit,
+        Beschwerden an die Online-Streitbeilegungsplattform der EU zu richten:
+        <a
+          href="http://ec.europa.eu/odr/"
+          class="underline decoration-solid underline-offset-4"
+          >http://ec.europa.eu/odr</a
+        >. Sie können allfällige Beschwerde auch an die oben angegebene
+        E-Mail-Adresse richten.
+      </p>
+      <p>
+        <b>Urheberrecht:</b> Die Inhalte dieser Webseite unterliegen, soweit
+        dies rechtlich möglich ist, diversen Schutzrechten (z.B dem
+        Urheberrecht). Jegliche Verwendung/Verbreitung von bereitgestelltem
+        Material, welche urheberrechtlich untersagt ist, bedarf schriftlicher
+        Zustimmung des Webseitenbetreibers.
+      </p>
+      <p>
+        <b>Haftungsausschluss:</b> Trotz sorgfältiger inhaltlicher Kontrolle
+        übernimmt der Webseitenbetreiber dieser Webseite keine Haftung für die
+        Inhalte externer Links. Für den Inhalt der verlinkten Seiten sind
+        ausschließlich deren Betreiber verantwortlich. Sollten Sie dennoch auf
+        ausgehende Links aufmerksam werden, welche auf eine Webseite mit
+        rechtswidriger Tätigkeit/Information verweisen, ersuchen wir um
+        dementsprechenden Hinweis, um diese nach § 17 Abs. 2 ECG umgehend zu
+        entfernen.<br />Die Urheberrechte Dritter werden vom Betreiber dieser
+        Webseite mit größter Sorgfalt beachtet. Sollten Sie trotzdem auf eine
+        Urheberrechtsverletzung aufmerksam werden, bitten wir um einen
+        entsprechenden Hinweis. Bei Bekanntwerden derartiger Rechtsverletzungen
+        werden wir den betroffenen Inhalt umgehend entfernen.
+      </p>
+      <p>
+        Quelle:
+        <a
+          href="https://www.ortner-rechtsanwalt.at/strafrecht-festnahme-verhaftung/"
```

Comment:
Woher kommt der random Link zu einer Rechtsanwaltkanzlei über Strafrecht?

Reply:
Uiuiui jetzt lernen wir, wie wir ein Impressum machen in WIR und du verwendest noch immer einen Generator dafür...